tests/integration/test_small_deployment_upgrades_edge_channel.py

```diff
@@ -51,7 +50,7 @@ async def c_writes_runner(ops_test: OpsTest, c_writes: ContinuousWrites):
 
 
 @pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "xlarge"])
-@pytest.mark.group(1)
+@pytest.mark.group("upgrade")
 @pytest.mark.abort_on_fail
 @pytest.mark.skip_if_deployed
 async def test_deploy_latest_from_channel(ops_test: OpsTest) -> None:
```

Comment:
For consistency with other tests:

```suggestion
async def test_build_and_deploy(ops_test: OpsTest) -> None:
```

Reply:
@Mehdi-Bendriss but we are not building the local charm, rather downloading from charmhub and deploying an "old" version of the cluster, then on further tests we upgrade to the locally built version.